analyses/sample-distribution-analysis/00-README.md

```diff
@@ -0,0 +1,31 @@
+# OpenPBTA sample-distribution-analysis
+
+## Folder content
+This folder contains scripts tasked to analyze the distribution of samples across cancer types and histologies in the PBTA dataset.  
+`00-install-packages.R` is a script written to install required packages and dependencies. This script is sourced in the other scripts.   
+`01-pbta-analysis-5` is a script written to perform the initial analyses on the PBTA dataset (as discussed in [issue #5](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/5)). This script produces TSV files containing the counts and percentages of samples across each unique cancer type, and the types sorted in order of descending expression at each unique brain location. 
+`02-multilayer-pie.R` is a script written to produce an interactive treemap and multilayer pie chart representing the distribution of samples across broad histologies, short histologies, and tumor types.  
+
+ The interactive treemap can be found [here](alexslemonade.github.io/OpenPBTA-analysis/histology-treemap.html).  
+ The interactive, multilayer pie chart can be found [here](alexslemonade.github.io/OpenPBTA-analysis/histology-pie.html).
+
+## Folder structure 
+
+The structure of this folder is as follows:
+
+```
```

Comment:
Not sure if you happened to make this manually. In case you did, I will share this tip that I've found helpful: you can install `tree` on your Mac with Homebrew (`brew install tree`) and run it for this directory ([ref](https://rschu.me/list-a-directory-with-tree-command-on-mac-os-x-3b2d4c4a4827?gi=a73423ee3eab)).